firefox/main.js

```diff
@@ -0,0 +1,22 @@
+var prefs = require('sdk/simple-prefs').prefs;
+
+var pageMod = require("page-mod");
+
+var self = require("self");
+
+var contentScripts = [
+  self.data.url("lib/p.js"),
+  self.data.url("lib/almond.js"),
+  self.data.url("content_scripts/visit.js"),
+  self.data.url("content_scripts/main.js"),
+];
+
+console.log("asdadasdiaskldjkasjdkljasdjklalkjdaksjsd");
```

Comment:
Seems fitting I just posted this, huh: https://github.com/mozilla/fxa-auth-server/pull/873#issuecomment-74368064